docs/fundamentals/syslib-diagnostics/syslib5003.md

```diff
@@ -0,0 +1,23 @@
+---
+title: SYSLIB5003 warning
+description: Learn about the diagnostic that generates compile-time warning SYSLIB5003.
+ms.date: 11/01/2024
+f1_keywords:
+  - syslib5003
+---
+
+# SYSLIB5003: SVE is a preview feature can be used by enabling EnablePreviewFeatures flag
+
+In .NET 9 first set of non-streaming SVE APIs were introduced as [[Experimental]](https://learn.microsoft.com/en-us/dotnet/api/system.diagnostics.codeanalysis.experimentalattribute?view=net-9.0). This indicates that both the internal implementation and the generated code for SVE may undergo changes. This includes potential modifications to method signatures, parameters, or namespaces in future updates, aimed at ensuring robust support for upcoming SVE technologies and streaming SVE designs. If you are using these APIs in your project, enable the preview mode in your project using `EnablePreviewFeatures`.
```

Comment:
We can use `<xref />` links, and we also need to make sure not to include `en-us` in any links. I think these APIs can be used without `EnablePreviewFeatures` too, since they don't require preview runtime bits--just suppressing the SYSLIB5003 diagnostic.

```suggestion
In .NET 9, the first set of non-streaming SVE APIs were introduced and annotated with the <xref:System.Diagnostics.CodeAnalysis.ExperimentalAttribute>. This indicates that both the internal implementation and the generated code for SVE may undergo changes. This includes potential modifications to method signatures, parameters, or namespaces in future updates, aimed at ensuring robust support for upcoming SVE technologies and streaming SVE designs. If you are using these APIs in your project, the `SYSLIB5003` diagnostic will need to be suppressed.
```